pages/linux/umask.md

```diff
@@ -0,0 +1,15 @@
+# umask
+
+> Display file creation mask or set file creation mask to octal value.
+
+- Display current value as octal:
+
+`umask`
+
+- Display current value symbolically:
+
+`umask -S`
+
+- Turn off permission bits with octal value 'nnnn':
```

Comment:
By "turn off" I mean the following example:
```
[root@latitude /]# umask
0022
[root@latitude /]# mkdir test
drwxr-xr-x    2 root root  4096 Apr 13 21:13 test
[root@latitude /]# umask 0777
[root@latitude /]# mkdir test2
d---------    2 root root  4096 Apr 13 21:14 test2

```

Please find updated commit with corrections.